lib/google/gapic/templates/README.md

```diff
@@ -0,0 +1,57 @@
+# Templates
```

Comment:
Should we put templates in a separate "toplevel" directory (not under lib)?

Reply:
Eventually I would like to see this configurable somehow. Like, perhaps setting the path in an environment variable so the output could be customized.

Reply:
+1 to all this. @landrito and I also chatted about allowing users to override individual parts of the templates so you don't need a whole set of alternative templates to, say, just change the license. 

I think we can tackle all of these ideas in followup PRs though.